csharp-basics/exercises/Scooters/Scooters/ScooterService.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections.Generic;
+
+namespace Scooters
+{
+    public class ScooterService : IScooterService
+    {
+        private static Dictionary<string, Scooter> _scooterList = new Dictionary<string, Scooter>();
+
+        public ScooterService()
+        {
+        }
+
+        public void AddScooter(string id, decimal pricePerMinute)
+        {
+            if (!_scooterList.ContainsKey(id))
```

Comment:
šis if daudz atkārtojas... var iznest metodē get by Id manuprāt un tad izmantot šo metodi. :)